### PR TITLE
[DO-2232]Added Tooltip, enabled selection less then minimum columns

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: src/const/testing.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 // FIXME: temporary variables for hiding functional
 export const HIDE_EXPORT_BUTTON = true;
 export const HIDE_FILTERS = true;
-export const HIDE_SELECT_COLUMNS = true;
+export const HIDE_SELECT_COLUMNS = false;
```
Comment (roman-tiukh) [inline src/const/testing.js:4]: here

File: src/components/table/SelectColumns.jsx [hunk 2]
```diff
@@ -106,11 +107,26 @@ const SelectColumns = (props) => {
...
+            <Tooltip
+              className="text-gray-500"
+              content={t('tableMinimumColumns')}
+              position="top"
+            >
+              <Button
+                className="w-32 h-8 mr-2 mb-2 bg-blue-700 cursor-pointer"
+                disabled={lessThenMinimumColumns}
+              >{t('apply')}
+              </Button>
```
Comment (roman-tiukh) [inline src/components/table/SelectColumns.jsx:120]:
```suggestion
              <Button
                className="w-32 h-8 mr-2 mb-2 bg-blue-700 cursor-pointer"
                disabled={lessThenMinimumColumns}
              >
                {t('apply')}
              </Button>
```

Comment (IegorSoboliev) [reply src/components/table/SelectColumns.jsx:120]: Done

File: src/const/testing.js (hunk 1 above)
Comment (IegorSoboliev) [reply src/const/testing.js:4]: planned to restore that before merging). Restored

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**